testing/efl/APKBUILD

```diff
@@ -13,55 +13,100 @@ depends_dev="giflib-dev gstreamer-dev gst-plugins-base-dev libpng-dev
 	libxscrnsaver-dev xorgproto luajit-dev"
 makedepends="$depends_dev dbus-dev openssl-dev fontconfig-dev freetype-dev bash
 	fribidi-dev zlib-dev"
+depends="dbus \
+	fontconfig \
+	freetype \
+	fribidi \
+	giflib \
+	gstreamer \
+	gst-plugins-base \
+	harfbuzz \
+	libpng \
+	libjpeg-turbo \
+	libx11 \
+	libxcomposite \
+	libxcursor \
+	libxdamage \
+	libxinerama \
+	libxrandr \
+	libxtst \
+	libxscrnsaver \
+	libxkbcommon \
+	libsndfile \
+	librsvg \
+	mesa-gles \
+	luajit \
+	wayland \
+	openssl"
```

Comment:
Most if not all of these are autodetected by abuild. Please see the build output for that.